pkg/app/web/src/types/module.ts

```diff
@@ -0,0 +1 @@
+export type LoadingStatus = "idle" | "loading" | "succeeded" | "failed";
```

Comment:
nit, sorry for the dumb question but I don't understand when do we need "idle" state 🤔

Reply:
@khanhtc1202 
`idle` is used to represent the state before fetching.

https://redux.js.org/tutorials/essentials/part-5-async-logic#loading-state-for-requests

It is not used currently, but it will be used in other modules/components.

Reply:
ah, got it, thank you 🙏